tilings/tiling.py

```diff
@@ -395,16 +390,10 @@ def split_16bit(n) -> Tuple[int, int]:
         return res.tobytes()
 
     @classmethod
-    def from_bytes(
-        cls,
-        arrbytes: bytes,
-        remove_empty_rows_and_cols=False,
-        derive_empty=False,
-        simplify=False,
-        sorted_input=True,
-    ) -> "Tiling":
```

Comment:
I don't like removing the arguments. Explicit is better than implicit. It makes the code less readable and you need to actually read the code to know how to use it. Putting the argument in the docstring would help but still having them is the best.

Maybe better is to realize that we don't need those arguments since we can assume that when we call `to_bytes` the tiling has been properly initialized.

Reply:
I agree with your solution, I at least never used another setting.